app/bloom/app.yaml

```diff
@@ -13,9 +13,9 @@ spec:
 
   # Source of the application manifests
   source:
-    repoURL: https://github.com/quickpipes/d4g-prod
-    targetRevision: HEAD
-    path: hello
+    repoURL: https://github.com/dataforgoodfr/d4g-prod
+    targetRevision: rs/bloom
```

Comment:
This is perfect for testing. We'll need to change this back to HEAD before merging (if this branch disappears your argo app will be unsyncable)